packages/plugins/EVM/src/state/Hub/hub.ts

```diff
@@ -118,7 +119,8 @@ class Hub implements EVM_Hub {
         })
         try {
             const isEIP1559 = chainResolver.isSupport(options.chainId, 'EIP1559')
-            if (isEIP1559) return await MetaSwap.getGasOptions(options.chainId)
+            if (isEIP1559 && chainId !== 592) return await MetaSwap.getGasOptions(options.chainId)
+            if (chainId === 592) return await AstarGas.getGasOptions(options.chainId)
```

Comment:
Move this first place. Meanwhile, please avoid using magic code.

Reply:
Ok